templates/scripts/add-vpn-user.sh

```diff
@@ -50,7 +66,11 @@ echo
 echo "After this is done you should be able to open the wireguard client and activate the tunnel."
 echo "You can download the client at https://www.wireguard.com/install/"
 echo
-echo "When it is running you should be able to access internal resources, e.g. mysql -h 10.10.10.123"
+echo "When it is running you should be able to access internal resources, eg. database server:" 
+echo "  with Kubernetes cluster DNS, run:"
```

Comment:
Let's not show both of these, it will just confuse people who don't have enough context. We can just have an example like <aws rds hostname>
If anything, you could mention that you can connect to things inside both the VPC and the kubernetes cluster.

Reply:
sure, good